openadapt/config.py

```diff
@@ -41,7 +41,8 @@
     "SCRUB_CHAR": "*",
     "SCRUB_LANGUAGE": "en",
     # TODO support lists in getenv_fallback
-    "SCRUB_FILL_COLOR": (255, 0, 0),
+    "TEST_SCRUB_FILL_COLOR": (255, 0, 0),
```

Comment:
I don't think we need this in the config file, we should just make this local to the test file as there shouldn't really be any reason to change the test colour. Also, could you please explain why we use (255,) instead of (255,0,0) now? Do you think it'd work better with the colour in hex (0xFF0000)?

Reply:
https://github.com/MLDSAI/OpenAdapt/pull/259#discussion_r1229969546

Reply:
Agreed!